RELEASE_NOTES

```diff
@@ -3,7 +3,34 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyblocks provides 169 software-specific easyblocks and 29 generic easyblocks.
+The latest version of easybuild-easyblocks provides 174 software-specific easyblocks and 29 generic easyblocks.
+
+
+v2.8.2 (July 12th 2016)
+-----------------------
+
+bugfix release
+- new easyblocks for 5 software packages that require customized support:
+    Amber (#958), Extrae (#955), Gurobi (#962), Paraver (#956), Tau (#887)
```

Comment:
great, very strong progress in this release on performance tools!

lgtm overall